code/obj/item/clothing/shoes.dm

```diff
@@ -544,3 +544,15 @@
 	name = "jester's shoes"
 	desc = "The shoes of a not-so-funny-clown."
 	icon_state = "jester"
+
+/obj/item/clothing/shoes/scream
+	name = "scream shoes"
+	icon_state = "pink"
+	step_sound = "scream"
+	desc = "AAAAAAAAAAAAAAAAAAAAAAA"
+	
+/obj/item/clothing/shoes/fart
+	name = "fart-flops"
+	icon_state = "tourist"
+	step_sound = "fart"
```

Comment:
While this'd definitely work, maybe perhaps the shoes could inherit their owner's noise preferences.

This could be done by putting in a proc that fires on-equip, and in that proc have it copy over their fart or scream settings from their appearance holder.

Reply:
You're welcome to help with this aspect

Reply:
I found that steps can take lists. This should be easy to fix